apiv6/generate_server.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Generate golang server stub from OpenAPI 3.0.0 spec
+
+
+function stage() {
+    BOLD_BLUE="\e[1m\e[34m"
+    RESET="\e[0m"
+    msg="$1"
+
+    echo
+    echo -e "$BOLD_BLUE$msg$RESET"
+}
+
+function generate_golang_server_from_openapi3_spec() {
+    stage "Generating golang server from openapi spec"
+
+    spec_file="$1"
+    package="$2"
+    output_dir="$3"
+
+    docker run --rm  -v "$(pwd):/local" \
+        openapitools/openapi-generator-cli generate  \
+        -i "/local/$spec_file" \
+        -g go-server \
+        --package-name "$package" \
+        -o "/local/$output_dir"
+
+
+    echo "Changing ownership of generated content to $USER:$USER"
+    sudo chown  -R "$USER:$USER"  "$output_dir" # by default the generated output is in user:group root:root
+    
+    rm "$output_dir/go.mod"
+    rm "$output_dir/README.md"
+
+    echo "Done"
+}
+
+# GENERATE CLOUDEXPORT
+cloudexport_package="cloudexportstub"
+cloudexport_spec="cloud_export.openapi.yaml"
+cloudexport_server_output_dir="localhost_apiserver/cloud_export"
```

Comment:
```suggestion
cloudexport_server_output_dir="stub_api_server/cloud_export"
```
This is subjective, but IMO the stub prefix is valuable information, and "localhost" is not.

Reply:
We can change the name, just remember to also  update cloudexport provider repo contents as it depends on this localhost_apiserver

Reply:
The server stubs for synthetics and cloudexport are generated separately as they come from separate swagger api secs, but I will make localhost_apiserver support both cloudexport and synthetics, not sure if this is what you mean?

Reply:
Yes, that is what I mean.